packages/xo-web/src/xo-app/backup-ng/new/metadata/index.js

```diff
@@ -169,6 +180,20 @@ export default decorate([
       setSettings: (_, _settings) => () => ({
         _settings,
       }),
+      setGlobalSettings: ({ setSettings }, name, value) => ({
+        settings = {},
+      }) => {
+        setSettings({
+          ...settings,
+          [GLOBAL_SETTING_KEY]: {
+            ...settings[GLOBAL_SETTING_KEY],
+            [name]: value,
+          },
+        })
+      },
+      setReportWhen({ setGlobalSettings }, { value }) {
+        setGlobalSettings('reportWhen', value)
```

Comment:
first parameter it's an effect.

Reply:
The first parameter of `setGlobalSettings` is an effect not a setting name.

Reply:
We don't pass the list of the effects to the effect. They are automatically injected.

Reply:
But here  `setGlobalSettings`  takes setting name as an effect when you call `setGlobalSettings`.

Reply:
No, please see the [documentation](https://github.com/JsCommunity/reaclette).